### PR TITLE
Test and fixes for CpxCNN

### PR DESCRIPTION
**Pull request by tszoldra**
Commit [#](https://github.com/markusschmitt/vmc_jax/commit/adf035b29417db4479e2021c8bb1275ff2ffa4cb) introduced different init_args for the nn.Conv function. Instead of passing param_dtype=... kwarg, the dtype=... kwargs was passed. This somehow set the kernel dtype to complex, but the biases remained float32. In this PR the kwargs passing is handled by init_fn_args like before. 
A test for CpxCNN is also added. It checks if the dtype of kernel and biases is complex as expected, as well as replicates the test for real CNNs. It turned out that in CpxCNN the padding=... argument for nn.Conv was missing, making the test fail, which was also fixed.

### CODE REVIEW

**Comment by tszoldra** [comment]
This error is caused by the latex \langle expression in the docstring. Possible fixes: 

1. Add 'r' before every docstring in the whole project (convert to raw strings).
2. Change every latex expression to include two \\, i.e. \langle -> \\langle.

In both cases, refactor of the whole documentation is needed for python=3.12 compatibility :(.

**Comment by markusschmitt** [comment]
I fixed the docstrings and merged it into the master branch. Can you try, whether merging it into this PR fixes it?
